### PR TITLE
External links PWA page

### PR DESCRIPTION
**Pull request by AndiAJ**
New external links test page.

Used the page created on my repo https://andiaj.github.io/testapp/externalLinks and works properly in the newly created smoke tests: `externalLinkPWATest` `emailLinkPWATest` `telephoneLinkPWATest`

• The PWA is properly installed
• The external links are properly identified and clicked
• The external links are properly opened

### CODE REVIEW

File: externalLinks.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      <link href="https://fonts.googleapis.com/css?family=Hanalei+Fill" rel="stylesheet">
+  </head>
+
+  <p class="header">Misc Link Types</p>
+  <section>
+    <a href="http://google.com">External link</a>
+  </section>
+
+  <section>
+    <a href="mailto://notta-r3al-3mail@gmail.com">Email link</a>
```
Comment (AaronMT) [inline externalLinks.html]: Lets use `example@example.com`

File: externalLinks.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  </section>
+
+  <section>
+    <a href="tel://+1-234-555-1212">Telephone link</a>
```
Comment (AaronMT) [inline externalLinks.html]: Let's use `tel://1234567890`

**Review by AaronMT — CHANGES_REQUESTED** [review]

File: externalLinks.html (hunk 2 above)
Comment (AndiAJ) [reply externalLinks.html]: Done and works properly.

File: externalLinks.html (hunk 1 above)
Comment (AndiAJ) [reply externalLinks.html]: Done and works properly.

**Review by AaronMT — APPROVED** [review]

**Pull request merged**